lib/ansible/modules/source_control/gitlab_service.py

```diff
@@ -0,0 +1,377 @@
+        module.fail_json(msg='Failed to connect to GitLab server: %s' % to_native(e))
+
+    try:
+        remote_service = project.services.get(service)
+    except gitlab.GitlabGetError as e:
+        module.fail_json(msg='No such a service %s' % service, exception=to_native(e))
+
+    ServicesHelper = GitLabServices(module, git)
+    if state == 'absent':
+        if not remote_service or not remote_service.created_at:
+            module.exit_json(changed=False, service={}, msg='Service not found', details='Service %s not found' % service)
+        else:
+            if module.check_mode:
+                module.exit_json(changed=True, service=remote_service.attributes)
+            try:
+                remote_service.delete()
+            except (gitlab.GitlabHttpError, gitlab.GitlabDeleteError) as e:
+                module.fail_json(msg='Failed to remove service %s' % service, exception=to_native(e))
+            else:
+                module.exit_json(changed=True, service=remote_service.attributes, result='Successfully deleted service %s' % service)
+
+    else:
+        if remote_service.created_at:
+            # update
+            try:
+                diff = {'before': str(remote_service.attributes)}
+                h = ServicesHelper.update(project, remote_service, active, params, events)
+                # if diff: fetch again
+                # diff['after'] = str(remote_service.attributes)
+            except gitlab.GitlabUpdateError as e:
+                module.fail_json(changed=False, msg='Could not update service %s' % service, exception=to_native(e))
+            else:
+                if h:
+                    module.exit_json(changed=True, service=remote_service.attributes, diff=diff, state='changed',
+                                     result='Successfully updated service %s' % service)
+                else:
+                    module.exit_json(changed=False)
+
+        else:
+            # create (could only happen once for this service during the life of the project)
+            # merge new attributes in the object retrieved from the server
+            local_service = ServicesHelper.asApiObject(active, params, events)
+            for k, v in local_service.items():
+                setattr(remote_service, k, v)
+            if module.check_mode:
+                module.exit_json(changed=True)
+            try:
+                remote_service.save()
+                diff = {'before': {}, 'after': str(remote_service.attributes)}
+            except (gitlab.GitlabCreateError, gitlab.GitlabUpdateError) as e:
+                module.fail_json(changed=False, msg='Could not create service %s' % service, exception=to_native(e))
+            else:
+                module.exit_json(changed=True, service=remote_service.attributes, diff=diff, state='created',
+                                 result='Successfully created service %s' % service)
```

Comment:
Use `msg` instead of `result` attribute
```suggestion
                                 msg='Successfully created service %s' % service)
```